lib/broadside/cli/gli_commands.rb

```diff
@@ -0,0 +1,142 @@
+def add_tag_flag(cmd)
+  cmd.desc 'Docker tag for application container'
+  cmd.arg_name 'TAG'
+  cmd.flag [:tag]
+end
+
+def add_target_flag(cmd)
+  cmd.desc 'Deployment target to use, e.g. production_web'
+  cmd.arg_name 'TARGET'
+  cmd.flag [:t, :target], type: Symbol, required: true
+end
+
+def add_instance_flag(cmd)
```

Comment:
well, i mean you would see the call to `add_target_flag` in `add_instance_flag`.  

As long as 100% of the time that you need an instance you also need a target - which i think is true? - then it's a little DRYer. i do see what you mean though; maybe the solution is to rename `add_instance_flag` to `add_instance_flag_and_target`.  then it will be clear.